### ISSUE TITLE
Without preview

### ISSUE THREAD

**Issue by emiliane** [vanevery/JavaCV-0.5-Stream-Test#1]
Can I send video to the server without preview?

**Comment by vanevery** [comment]
No, I don't think you can.  This application depends on the preview frames being generated (and probably shown)

**Comment by IgeNiaI** [comment]
It's not possible because of android api, but you can make preview view invisible or with 1 px width and height.
